
package com.sforce.soap.partner;

import javax.xml.ws.WebFault;


/**
 * This class was generated by Apache CXF 2.2.12
 * Thu Sep 01 08:54:15 CDT 2011
 * Generated source version: 2.2.12
 * 
 */

@WebFault(name = "InvalidNewPasswordFault", targetNamespace = "urn:fault.partner.soap.sforce.com")
public class InvalidNewPasswordFault extends Exception {
    public static final long serialVersionUID = 20110901085415L;
    
    private com.sforce.soap.partner.fault.InvalidNewPasswordFault invalidNewPasswordFault;

    public InvalidNewPasswordFault() {
        super();
    }
    
    public InvalidNewPasswordFault(String message) {
        super(message);
    }
    
    public InvalidNewPasswordFault(String message, Throwable cause) {
        super(message, cause);
    }

    public InvalidNewPasswordFault(String message, com.sforce.soap.partner.fault.InvalidNewPasswordFault invalidNewPasswordFault) {
        super(message);
        this.invalidNewPasswordFault = invalidNewPasswordFault;
    }

    public InvalidNewPasswordFault(String message, com.sforce.soap.partner.fault.InvalidNewPasswordFault invalidNewPasswordFault, Throwable cause) {
        super(message, cause);
        this.invalidNewPasswordFault = invalidNewPasswordFault;
    }

    public com.sforce.soap.partner.fault.InvalidNewPasswordFault getFaultInfo() {
        return this.invalidNewPasswordFault;
    }
}
